### ISSUE TITLE
Upgrade mapbox-gl to 1.10.0

### ISSUE THREAD

**Issue by IzikLisbon** [WiredSolutions/react-azure-maps#58]
The current version of mapbox-gl in package.json (^1.8.1) includes few security risks due to an old version of minimist (version 0.0.8). Upgrading to the latest 1.10.0 will solve it.

from https://github.com/mapbox/mapbox-gl-js/releases/tag/v1.10.0:
```
Upgrade minimist to ^1.2.5 to get fix for security issue CVE-2020-7598 upstream (#9425, fixed by #9425)
```

Since anyway package.json is marked to auto-accept minor releases (^), I assume this should be a minor change.

**Comment by msasinowski** [comment]
@IzikLisbon Thank you, it will be included in the next minor release  0.1.1 @srednicki95 
https://github.com/WiredSolutions/react-azure-maps/pull/new/chore/bump-mapbox-gl

**Comment by izikl** [comment]
@msasinowski Thank You!
